docs/molgenis/run_docker.md

```diff
@@ -1,34 +1,126 @@
 # Run using docker compose
 
-Steps:
+## Requirements
 
 * Install [Docker compose](https://docs.docker.com/compose/install/).
-* Download
-  molgenis-emx2 <a href="https://raw.githubusercontent.com/mswertz/molgenis-emx2/master/docker-compose.yml" download>
-  docker-compose.yml</a> file
-* In directory with docker-compose.yml run:
+* Install [Git](https://github.com/git-guides/install-git).
 
-  ```
-  docker-compose up
-  ```
+### Clone emx2 repository
 
-* Open on http://localhost:8080
+This will download the latest repository of emx2.
 
-Useful commands:
+`git clone https://github.com/molgenis/molgenis-emx2.git`
```

Comment:
dont need the whole repo for running via compose, could you clone / down load part of the repo ? or could we put all of the compose stuff in a single file ?

Reply:
yes that would be ideal. Then we could even add the compose to releases.